packages/sdk-react/src/components/onboarding/OnboardingFile/OnboardingFileUploader/OnboardingFileUploader.tsx

```diff
@@ -71,17 +71,17 @@ export const OnboardingFileUploader = ({
           size="large"
           variant="contained"
           color="primary"
-          disabled={isLoading}
+          disabled={isPending}
           startIcon={
-            isLoading ? (
+            isPending ? (
               <CircularProgress color="inherit" size={22} />
             ) : (
               <CloudUpload />
             )
           }
         >
-          {isLoading && t(i18n)`Processing...`}
-          {!isLoading && t(i18n)`Choose document`}
+          {isPending && t(i18n)`Processing...`}
+          {!isPending && t(i18n)`Choose document`}
```

Comment:
`isPending` will be false when `isError` is `true`, would be better to use `isSuccess` instead